parl/core/fluid/agent.py

```diff
@@ -26,11 +26,49 @@
 
 
 class Agent(AgentBase):
+    """
+    | `alias`: ``parl.Agent``
+    | `alias`: ``parl.core.fluid.agent.Agent``
+
+    | Agent is one of the three basic classes of PARL. 
+
+    | It is responsible for interacting with the environment and collecting data for training the policy.
+    | To implement a customized ``Agent``, users can:
+
+      .. code-block:: python
+
+        import parl
+
+        class MyAgent(parl.Agent):
+            def __init__(self, algorithm, act_dim):
+                super(MyAgent, self).__init__(algorithm)
+                self.act_dim = act_dim
+      This class will initialize the neural network parameters automatically, and provides an executor for users to run the programs (self.fluid_executor).
+
+    Attributes:
+        gpu_id (int): deprecated. specify which GPU to be used. -1 if to use the CPU.
+        fluid_executor (fluid.Executor): executor for running programs of the agent.
+        alg (parl.algorithm): algorithm of this agent.
+
+    Public Functions:
+        - ``build_program`` (**abstract function**): build various programs for the agent to interact with outer environment.
+        - ``set_weights``: return a list containing all the parameters of self.alg.
```

Comment:
comments of `set_weights` and `get_weights` are misplaced.
And `get_weights` will return a dict.